src/ClusterBootstrap/cloud_init_aztools.py

```diff
@@ -17,15 +17,22 @@
     remove_nsg_rule_whitelist, \
     delete_nsg_rule_whitelist, \
     create_nsg_rules_with_service_tags, \
+    create_nsg_rule_with_service_tag, \
     delete_nsg_rules_with_service_tags, \
     create_logging_storage_account, \
     create_logging_container, \
     delete_logging_storage_account, \
     get_connection_string_for_logging_storage_account
 
 from cloud_init_deploy import load_node_list_by_role_from_config
+from ctl import run_kubectl
 sys.path.append("../utils")
-from ConfigUtils import add_configs_in_order
+from ConfigUtils import add_configs_in_order, merge_config
+
+ENV_CNF_YAML = 'config.yaml'
```

Comment:
Can we consistently use double quotes for strings?

Reply:
corrected